app/Http/Controllers/Gear/GearController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers\Gear;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\StoreGearRequest;
+use App\Models\Gear;
+use App\Services\Gear\GearService;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+
+class GearController extends Controller
+{
+    public function index(Request $request): View
+    {
+        $gear = Gear::query()
+            ->where('user_id', auth()->id())
+            ->orderByDesc('id')
```

Comment:
This should be `created_at`, since ID is a UUID. Probably worth indexing `created_at` for performance here.